lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/transport/netty/NettyMessagingTransport.java

```diff
@@ -130,10 +141,46 @@ private NettyMessagingTransport(
 
     final String host = UNKNOWN_HOST_NAME.equals(hostAddress) ? localAddressProvider.getLocalAddress() : hostAddress;
 
+    final SslContext sslContextClient;
+    final SslContext sslContextServer;
+
+    ssl = System.getProperty("ssl") != null;
```

Comment:
Oh, I misunderstood your saying! Currently, I couldn't find any details about System.getProperty("ssl") except Netty Http example I referenced. I'll find out is there any good implementation for support SSL. Thank you very much for pointing out this!